tool/teleport/common/teleport.go

```diff
@@ -107,8 +107,13 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	remoteForward := app.Command(teleport.RemoteForwardSubCommand, "Used internally by Teleport to re-exec itself to remote port forward.").Hidden()
 	checkHomeDir := app.Command(teleport.CheckHomeDirSubCommand, "Used internally by Teleport to re-exec itself to check access to a directory.").Hidden()
 	park := app.Command(teleport.ParkSubCommand, "Used internally by Teleport to re-exec itself to do nothing.").Hidden()
+	migrate := app.Command("migrate", "Migrate data between two backends.").Hidden()
```

Comment:
nit: `migrate` is a pretty broad term for a top-level command.  I'd lean toward something like either `backend migrate` or `migrate backend` so that we can introduce other backend and/or migration utilities without eating up our namespacing too much.